src/datadoc/frontend/fields/display_dataset.py

```diff
@@ -208,7 +208,7 @@ class DatasetIdentifiers(str, Enum):
     DatasetIdentifiers.KEYWORD: MetadataInputField(
         identifier=DatasetIdentifiers.KEYWORD.value,
         display_name="Nøkkelord",
-        description="Her kan en oppgi nøkkelord som beskriver datasettet, og som kan brukes i søk",
+        description="Her kan en oppgi nøkkelord som beskriver datasettet, og som kan brukes i søk. Nøkkelordene må legges inn som en komma separert streng. F.eks. befolkning, skatt, arbeidsledighet",
```

Comment:
I think "komma separert"  is one word, not two...
```suggestion
        description="Her kan en oppgi nøkkelord som beskriver datasettet, og som kan brukes i søk. Nøkkelordene må legges inn som en kommaseparert streng. F.eks. befolkning, skatt, arbeidsledighet",
```